src/core/expressionvariablemodel.cpp

```diff
@@ -90,6 +91,21 @@ void ExpressionVariableModel::reloadVariables()
     }
   }
 
+  // Second add readonly project variables
+  QVariantMap projectVariables = ExpressionContextUtils::projectVariables( mCurrentProject );
+  for ( const QString &varName : projectVariables.keys() )
+  {
+    QStandardItem *nameItem = new QStandardItem( varName );
+    QVariant varValue = projectVariables.value( varName ).toString();
+
+    nameItem->setData( varName, VariableName );
+    nameItem->setData( varValue, VariableValue );
```

Comment:
Now that we have both application and project variables, we need to add a role to identify the scope.

I'd go for a VariableScope role, then add a new VariableScopes enum with { ApplicationScope, ProjectScope }, and use that new enum as value for that role.

Reply:
Oh and while we're at it, let's add a VeriableEditable role and get rid of the model's Q_INVOKVABLE isEditable function. It'll be much nicer.

Reply:
Done. :+1: